### PR TITLE
Stories/cg/add i18n module

### LINKED ISSUE #95: Add yoast i18n module
The yoast i18n module allows you to add a call to action to your settings pages inviting users to translate your plugin if it is not available in their language yet. We should include the module on the WooCommerce SEO settings page as well.

`yoast_i18n` is included in Yoast SEO core so you can simply expect it to be there.

### PR DESCRIPTION
**Pull request by CarolineGeven**
Fixes #95 

Currently the Translation box for the addons is always showing the Yoast SEO translation box, if Yoast SEO itself isn't fully translated yet in the language that is selected.

Ideally we want to show the Yoast for Woo Translation box, as the translation box will not be shown if Yoast SEO is fully translated, even though Yoast for Woo isn't yet (fully) translated, as shown below. This pull fixes that. 

schermafbeelding 2016-03-03 om 17 34 28

### For acceptance testing
Languages to test with:
Yoast SEO fully translated - Woo SEO partially/not: choose Catalan for language.
Yoast SEO not fully translated - Woo SEO not fully translated: choose for example Nederlands (Formeel)

Be sure pr https://github.com/Yoast/wordpress-seo/pull/4600 is merged before merging this pull.

### CODE REVIEW

File: wpseo-woocommerce.php [hunk 1]
```diff
@@ -854,14 +873,20 @@ public static function admin_header( $form = true, $option_long_name = 'yoast_wp
 	 * @return mixed
 	 */
 	public static function admin_footer( $submit = true, $show_sidebar = true ) {
+		// By removing the action 'wpseo_admin_footer' we make sure the Yoast SEO i18n module isn't loaded.
+		remove_all_actions( 'wpseo_admin_footer' );
```
Comment (omarreiss) [inline wpseo-woocommerce.php]: This should be fixed in wordpress seo en not hacked around here.

**Comment by omarreiss** [comment]
We need to make sure wordpress seo doesn't include the i18n module for itself on add-on pages before this can be merged.

**Comment by atimmer** [comment]
CR Done 👍🏻

**Comment by terw-dan** [comment]
Acceptance ok, but the build has errors.

**Comment by terw-dan** [comment]
The build is still failing

**Pull request merged**